examples/controls_example/public/react_controls/control_group/get_control_group_factory.tsx

```diff
@@ -167,6 +169,19 @@ export const getControlGroupEmbeddableFactory = (services: {
         dataViews.next(newDataViews)
       );
 
+      /** Fetch the allowExpensiveQuries setting for the children to use if necessary */
+      try {
+        const result = await services.core.http.get<{
+          allowExpensiveQueries: boolean;
+          // TODO: Rename this route as part of https://github.com/elastic/kibana/issues/174961
+        }>('/internal/controls/optionsList/getExpensiveQueriesSetting', {
+          version: '1',
+        });
+        allowExpensiveQueries$.next(result.allowExpensiveQueries);
+      } catch {
+        allowExpensiveQueries$.next(true); // default to true on error
```

Comment:
Why do you default to true on error but the initial value is false? Shouldn't the error and initial value be the same?

Reply:
Good call - I remember flip-flopping back and forth on this one but I don't remember why I settled on `false` tbh 🤷 Done in 51e2aaef9bac40308d27ccc75ae1d5016e8397aa